packages/bot/src/helpers.ts

```diff
@@ -232,11 +244,23 @@ export function createBotHelpers(bot: Bot): BotHelpers {
     getApplicationInfo: async () => {
       return bot.transformers.application(bot, snakelize(await bot.rest.getApplicationInfo()))
     },
-    getApplicationCommandPermission: async (guildId, commandId) => {
-      return bot.transformers.applicationCommandPermission(bot, snakelize(await bot.rest.getApplicationCommandPermission(guildId, commandId)))
+    getCurrentAuthenticationInfo: async (bearerToken) => {
+      return await bot.rest.getCurrentAuthenticationInfo(bearerToken)
     },
-    getApplicationCommandPermissions: async (guildId) => {
-      return (await bot.rest.getApplicationCommandPermissions(guildId)).map((res) =>
+    exchangeToken: async (options) => {
+      return await bot.rest.exchangeToken(options)
+    },
+    revokeToken: async (options) => {
+      return await bot.rest.revokeToken(options)
+    },
+    getApplicationCommandPermission: async (guildId, commandId, options) => {
+      return bot.transformers.applicationCommandPermission(
+        bot,
+        snakelize(await bot.rest.getApplicationCommandPermission(guildId, commandId, options)),
```

Comment:
Not sure why the snakelize is here. I dont mean that its unnecessary just trying to understand the logic/reasoning here. Perhaps ur right and it is necessary

Reply:
I added it as the `rest.get` method calls the camelize method, and in the [transformer for the application command](<https://github.com/discordeno/discordeno/blob/6ea1360a07e72aa7d7f7742ec1a196b6c391d800/packages/bot/src/transformers/applicationCommandPermission.ts>) it uses the `application_id` as snake_case, so it would get undefined

Reply:
Doesn't createBot enable the snake case option by default now? The reason i ask is looking at this code, im thinking is this wasted CPU by running snakelize on something thats already snakelized

Reply:
I will check at runtime what actually happens later

Reply:
It is in fact needed, 
![image](https://github.com/discordeno/discordeno/assets/45207244/6a6a5534-3553-4661-a283-0132df2e5aa2)

for context this is res and snakedRes
```ts
    getApplicationCommandPermission: async (guildId, commandId, options) => {
      const res = await bot.rest.getApplicationCommandPermission(guildId, commandId, options)
      const snakedRes = snakelize(res)

      return bot.transformers.applicationCommandPermission(bot, snakedRes)
    },
``` and the debugger is after the snakelize

\* i have censored the ids